tensorflow/python/ops/embedding_ops.py

```diff
@@ -1007,9 +1008,26 @@ def embedding_lookup_sparse_impl(
     name,
 ):
   """Implementation of sparse embedding aggregation."""
-  if len(params) == 1 and max_norm is None and allow_fast_lookup:
+  need_sparse_segment_gradient = False
+  # Ensure we can query the devices below.
+  segment_ids = ops.convert_to_tensor(segment_ids, name="segment_ids")
+  if (len(params) == 1 and
+      not isinstance(params[0], resource_variable_ops.BaseResourceVariable)):
+    params = [ops.convert_to_tensor(params[0], name="params")]
```

Comment:
We have some tests failing because of this line.
```
TypeError: Converting ShardedVariable to tensor in embedding lookup ops is disallowed.
```
For example, `//tensorflow/python/distribute:sharded_variable_test`

Reply:
Ack, will pull in and check on our end, thanks!